analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -319,10 +319,10 @@ class SpdxDocumentFile(
 
         // If the VCS information cannot be determined from the download location, fall back to try getting it from the
         // VCS working tree itself.
-        val vcs = getVcsInfo() ?: run {
+        val vcs = run {
```

Comment:
The `run` is no longer necessary. This can be simplified to
`val vcs = VersionControlSystem.forDirectory(packageDir)?.getInfo() ?: getVcsInfo().orEmpty()`